rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp

```diff
@@ -62,6 +69,47 @@ SequentialCompressionWriter::~SequentialCompressionWriter()
 
 void SequentialCompressionWriter::compression_thread_fn()
 {
+  if (compression_options_.thread_nice_value) {
+
+#ifdef _WIN32
+    ROSBAG2_COMPRESSION_LOG_WARN_STREAM(
+      "Lowering of process priority is not implemented for windows");
```

Comment:
does this depends on the application for lower or higher?

```suggestion
      "Changing process priority is not implemented for windows");
```

Reply:
I think we can add support for windows. I can contribute.